flutter/test/widgets_binding_observer_test.dart

```diff
@@ -26,6 +29,130 @@ void main() {
       flutterTrackingDisabledOptions.useNativeBreadcrumbTracking();
     });
 
+    testWidgets(
+        'app lifecycle does not generate new trace if SentryNavigatorObserver is used',
+        (WidgetTester tester) async {
+      Future<void> sendLifecycle(String event) async {
```

Comment:
If the `sendLifecycle` method is the same in every test, we would move it outside to remove duplication.